garrysmod/lua/includes/modules/widget.lua

```diff
@@ -28,19 +28,18 @@ local function UpdateHovered( pl, mv )
 
 	local OldHovered = pl:GetHoveredWidget()
 	pl:SetHoveredWidget( NULL )
-
+	
+	local eyePos = pl:EyePos()
 	local trace =
 	{
-		start	= pl:EyePos(),
-		endpos	= pl:GetAimVector(),
+		start	= eyePos,
+		endpos	= pl:GetAimVector():Mult( 256 ):Add( eyePos ),

```

Comment:
It needs to be done before the `local trace` and it's Vector.Mul, not Mult, also they don't return anything.